pkgs/by-name/sa/samrewritten/package.nix

```diff
@@ -11,7 +11,7 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "samrewritten";
-  version = "unstable-2023-05-23";
+  version = "202008-unstable-2023-05-22";
```

Comment:
This project uses `YYYYMM` tags, and puts `x.y` versions into its release descriptions… kind of upsetting :confused:.

But [nobody on repology uses the `x.y` versions](https://repology.org/project/samrewritten/versions), and Gentoo uses the `YYYYMM` tag number, so I guess we'll go with that.